### PR TITLE
Add SpicyButton

### PR DESCRIPTION
**Pull request by lukecrum**
Added SpicyButton
## Project URL

https://github.com/lukecrum/SpicyButton
## Description

 Added SpicyButton to the button category
## Checklist

- [X] Only one project/change is in this pull request
- [X] Addition in chronological order (bottom of category)
- [X] Supports iOS 9 / tvOS 10 or later
- [X] Supports Swift 4 or later
- [X] Has a commit from less than 2 years ago
- [X] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 1 link issue, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issue by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
1269 | [301](https://httpstatuses.com/301) | https://creativesdk.adobe.com/ redirects to<br>https://www.adobe.io/apis/creativecloud/creativesdk.html


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -1916,7 +1916,7 @@ Most of these are paid services, some have free tiers.
 * [JSButton](https://github.com/imjog/JSButton) - A fully customisable swift subclass on UIButton which allows you to create beautiful buttons without writing any line of code. 🔶
 * [TransitionButton](https://github.com/AladinWay/TransitionButton) - UIButton sublass for loading and transition animation :large_orange_diamond:
 * [ButtonProgressBar-iOS](https://github.com/thePsguy/ButtonProgressBar-iOS) - A small and flexible UIButton subclass with animated loading progress, and completion animation. :large_orange_diamond:
-
+* [SpicyButton](https://github.com/lukecrum/SpicyButton) - Full-featured IBDesignable UIButton class 🔶
```
Comment (lfarah) [inline README.md]: Missing a space between your lib and the **Calendar** category

**Comment by lfarah** [comment]
Hey @lukecrum, thanks for contributing! Please fix the issue above so we can Merge your PR 😉

**Comment by aaronjsutton** [comment]
Oh, I must’ve missed that. Apologies

**Comment by lukecrum** [comment]
fixed ;)

**Comment by lfarah** [comment]
No problem! Thanks for the quick fix

**Comment by lukecrum** [comment]
thanks for the quick merge!

**Pull request merged**